app/src/main/java/de/danoeh/antennapod/adapter/StatisticsListAdapter.java

```diff
@@ -0,0 +1,105 @@
+package de.danoeh.antennapod.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.joanzapata.iconify.widget.IconTextView;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.glide.ApGlideSettings;
+import de.danoeh.antennapod.core.storage.DBReader;
+
+/**
+ * Adapter for the statistics list
+ */
+public class StatisticsListAdapter extends BaseAdapter {
+    private Context context;
+    ArrayList<DBReader.StatisticsItem> feedTime = new ArrayList<>();
+
+    public StatisticsListAdapter(Context context) {
+        this.context = context;
+    }
+
+
+    @Override
+    public int getCount() {
+        return feedTime.size();
+    }
+
+    @Override
+    public DBReader.StatisticsItem getItem(int position) {
+        return feedTime.get(position);
+    }
+
+    @Override
+    public long getItemId(int position) {
+        return 0;
```

Comment:
you should return something useful here, e.g. ``feed.getId()``
See https://developer.android.com/intl/es/reference/android/support/v7/widget/RecyclerView.Adapter.html#setHasStableIds(boolean)